app/assets/stylesheets/content/_notifications.sass

```diff
@@ -28,33 +28,95 @@
 
 // NOTE: these are demo ski
 
+//nm - notification messages
+$nm-font-size: rem-calc(13px)
+$nm-border-radius: rem-calc(2px)
+$nm-box-padding: rem-calc(10px 35px 10px 35px)
+
+$nm-color-error-border: #ca3f3f
+$nm-color-error-icon: #ca3f3f
+$nm-color-error-background: #fedada
```

Comment:
I am unsure if we should define those variables (or similar ones) globally.

What we will encounter elsewhere too is the background color (e.g. for errorneous fields) and the border color (also for errorneous fields).

I can also imagine that we will use the same red/yellow/green tones for other kinds of error/success indication throughout the application.

I'd say we should at least provide the most basic colors (red/green as border/background) in the `_variables.sass`, and probably use them here by assigning them to the `$nm-*` variables.

However, note that I might be missing some important concept within our Sass files, so take my proposal with caution, I am not often working on the Sass side ^^